components/block/element.js

```diff
@@ -244,6 +252,26 @@ export default class Block extends LitElement {
       </main>
     `;
   }
+
+  showErrorModal() {
+    const body = `<div class="error_message">
```

Comment:
Fix formatting

Reply:
Fixed in 2bd69f54c37268ad510fd157e41b7d1234b9d95e